server/adminExists.go

```diff
@@ -0,0 +1,30 @@
+package server
+
+import (
+	"encoding/json"
+
+	"github.com/kuzzleio/sdk-go/types"
+)
+
+//AdminExists checks if an administrator account has been created, and return a boolean as a result.
+func (s *Server) AdminExists(options types.QueryOptions) (bool, error) {
+	result := make(chan *types.KuzzleResponse)
+
+	query := &types.KuzzleRequest{
+		Controller: "server",
+		Action:     "adminExists",
+	}
+
+	go s.k.Query(query, options, result)
+
+	res := <-result
+
+	if res.Error != nil {
+		return false, res.Error
+	}
+
+	var r bool
+	json.Unmarshal(res.Result, &r)
```

Comment:
I'm not familiar with Go, but shouldn't we get `res.Result.Exists` ?
cf. http://docs.kuzzle.io/api-documentation/controller-server/admin-exists/
Response from Kuzzle is:
```
{
 result: {
    exists: true/false
  }
}
```

Reply:
Hmm just to be sure to understand correcty:

In `now.go`, for example, to unmarchal the result value of `{result: {now: <timestamp>}}` and get the timestamp we have this:
```go
 type now struct {
   Now int `json:"now"`
 }

n := now{}
json.Unmarshal(res.Result, &n)
return n.Now, nil
```

For admin exists, to unmarshal `{result: {exists: <true|false>}}`, and get the value of `result.exists`, shouldn't we have rather following code ?
```go
 type exists struct {
   Exists bool `json:"exists"`
 }

r := exists{}
json.Unmarshal(res.Result, &r)
return r.Exists, nil
```
or is it full implicit ?

Reply:
That's right, exists is actually inside result.. my bad